package.json

```diff
@@ -61,8 +61,10 @@
   "jest": {
     "collectCoverageFrom": [
       "**/*.{js,jsx}",
-      "!./build/**",
-      "!./node_modules/**"
+      "!**/*.stories.jsx",
+      "!**/build/**",
+      "!**/coverage/**",
+      "!**/cypress/support/**"
```

Comment:
Rather than a blacklist at root, why don't we just have a whitelist of `src` and then just blacklist inside `src`? Don't need the above 3 lines then.

Should blacklist everything in `src/__test__`.